.github/workflows/release.yml

```diff
@@ -6,15 +6,21 @@ name: Release
 jobs:
   release-please:
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: write
+      pull-requests: write
```

Comment:
Just looking at doing similar things elsewhere.  Why were the `contents` and `pull-requests` write perms granted?  The [npm docs](https://docs.npmjs.com/generating-provenance-statements#publishing-packages-with-provenance-via-github-actions) say only `id-token` is needed.

Reply:
I found this useful - https://docs.github.com/en/rest/overview/permissions-required-for-github-apps?apiVersion=2022-11-28

Basically only `id-token` and `contents` need `write` perms.

Reply:
To answer my own question it looks like `contents` is needed to create releases and `pull-requests` is to create the release PR.

Reply:
That's the one. When I added the id token permission release-please started failing until i added the other perms.